src/sugar3/bundle/__init__.py

```diff
@@ -46,25 +47,31 @@
 
 Optional metadata keys are;
 
-* `summary` - a short summary of the activity that may be displayed in the List or Home Views,
+* `summary` - a short summary of the activity that may be displayed in
+  the List or Home Views,
 
 * `mime_types` - list of MIME types supported by the activity,
   separated by semicolons.  Your `read_file` method must be able to read
-  files of these MIME types.  Used to offer your activity when opening a
-  downloaded file or a journal object.
+  files of these MIME types.  Used to offer your activity in a list of activities
+  when opening a downloaded file from the web or a certain journal object.
 
 * `url` - link to the home page for the activity,
 
 * `repository` - link to repository for activity code, for use by git clone,
 
 * `single_instance` - if yes, only a single instance of an activity
-  should be started, and if another instance is requested the existing
-  instance shown,
+  should be started at any one time, and if another instance is requested
+  the existing instance shown,
 
-* `max_participants` - maximum number of participants for sharing an activity,
+* `max_participants` - maximum number of participants that can join a shared activity,
 
 * `tags` - more context into which to place an activity,
 
+* `category`/`categories` - define/choose a category/categories that fits your
+activity suitably. You can choose from the list available at http://activities.sugarlabs.org/en-US/sugar/,
```

Comment:
When you use <kbd>/</kbd> to separate terms, the meaning is ambiguous; it could mean and, or, or and/or.

Add a list of category names here rather than refer to an obsolete web service.

Define the format of the value; space or <kbd>;</kbd> delimited.

My preference is to use `tags`, with `category` and `categories` to be acceptable aliases.